src/algokit/cli/localnet.py

```diff
@@ -126,7 +128,26 @@ def config_command(*, engine: str | None, force: bool) -> None:
     "AlgoKit will not manage the configuration of named LocalNet instances, "
     "allowing developers to configure it in any way they need.",
 )
-def start_localnet(name: str | None) -> None:
+@click.option(
```

Comment:
Do we want to ensure that a user doesn't supply both `name` and `config_path`?

Reply:
Why do you think doing both is not an option?